Dockerfile.tf

```diff
@@ -0,0 +1,14 @@
+FROM tensorflow/tensorflow:2.8.0
```

Comment:
I think this should change to a single base image that can run both systems vs splitting them out...